packet/config.go

```diff
@@ -8,6 +8,7 @@ type Config struct {
 	ProjectID             string  `json:"projectId"`
 	BaseURL               *string `json:"base-url,omitempty"`
 	LoadBalancerConfigMap string  `json:"loadbalancer-configmap"`
+	ConfigMapEnable       bool    `json:"configmap-disable"`
```

Comment:
The field of the struct is `ConfigMapEnable`, but the json is `"configmap-disable"`?